cmd/bootstrap/README.md

```diff
@@ -87,65 +122,98 @@ Now that you have all your CRs placed on a local directory, just run:
 ```shell
 # To bootstrap your site using the binary
 export SKUPPER_PLATFORM=podman
-./bootstrap ./
+./bootstrap -p ./
 ```
 
+You can also use `-n=<name>` to override the namespace specified in the CRs.
+
 Alternatively you can also run the `bootstrap.sh` script:
 
 ```shell
 # To bootstrap your site using the shell script
 export SKUPPER_PLATFORM=podman
-./cmd/bootstrap/bootstrap.sh ./
+./cmd/bootstrap/bootstrap.sh -p ./
 ```
 
+Similarly to the binary, you can also use `-n <namespace>` to override the namespace
+defined in the source CRs.
+
 Remember that you can set the `SKUPPER_PLATFORM` environment variable to
 any of the platforms mentioned earlier.
 
+If you do not want to run an actual site from your CRs, but instead, you just
+need to produce a bundle that can be installed somewhere else, then you can
+use `bundle` or `tarball` as the bundle strategy (-b flag).
+
+The `bundle` strategy will produce a self-extracting shell archive that can
+be executed to install your local non-kubernetes site.
+
+If you are not comfortable executing the produced script, you can also choose
+`tarball` as the bundle strategy. It produces a tar ball that contains an
+`install.sh` script, that basically performs the same procedure of the
+self-extracting `bundle`.
+
 After the bootstrap procedure is completed, it will provide you some relevant
 information like:
 
-* Location where static tokens have been defined (when a `RouterAccess` is defined)
-* Location where the site bundle has been saved (when using `bundle` or `tarball`)
+* Location where static links have been defined (when a `RouterAccess` is provided)
+* Location where the site bundle has been saved (if bundle strategy flag `-b` set)
+* Namespace, Site name, Platform, Version and path to the sources
 
 #### Site bundles
 
 When a site bundle is produced, the bootstrap procedure will point you
 to the location where it has been saved.
 
+##### Bundle installation usage
+
+The bundle installation script accepts the following flags:
+
+```
+-h               help
+-p <platform>    podman, docker, systemd
+-n <namespace>   if not provided, the namespace defined in the bundle is used (if none, default is used)
+-x               remove site and namespace
+-d <directory>   dump static links into the provided directory 
+```
+
 #### Removing
 
-To remove your site, you can run a local script, that takes the site name
+To remove your site, you can run a local script, that takes the namespace
 as an argument. The platform is identified by the script so you don't need
 to export it.
 
+If the namespace is omitted, the "default" namespace is used. 
+
 ```shell
-./cmd/bootstrap/remove.sh <site-name>
+./cmd/bootstrap/remove.sh [namespace]
```

Comment:
```
$ ./bootstrap -p ~/test-podman -n east                                                                                                                  
Skupper nonkube bootstrap (version: nodeport-899-ge39ee03d)
Site "east" has been created on namespace "east"
Platform: podman
Definition is available at: /home/nluaces/.local/share/skupper/namespaces/east/sources
 

$ ./remove.sh east                                                                                                                       
east-skupper-router
Removed "/home/nluaces/.config/systemd/user/default.target.wants/skupper-east.service".
```
What does it mean `default.target.wants`? the directory `east` was deleted though.

Reply:
This is actually being displayed by systemctl when the systemd service is removed.
I have updated the `remove.sh` script to provide a better response message.